### ISSUE TITLE
How to Light Bloomery?

### ISSUE THREAD

**Issue by enderking152** [EnderiumSmith/CharcoalPit2#14]
I set up a bloomery in 1.16 but I can't figure out how to light it

**Comment by winnetrie83** [comment]
I have the same problem: 

I am able to place an iron ore into it with shift rightclicking, after that i can place several pieces of coal into it with rightclick.
However i can not light it up.

**Comment by EnderiumSmith** [comment]
it has to be a full block. also try shift right clicking

**Comment by winnetrie83** [comment]
Yes, i finally found how it works. Apparently after clicking in your coal/charcoal, you have to rightclick 3 more ores into it. After that it is a full block and can be lit up.

On the curseforge website, you described it that we have to shiftright click the ore into it then rightclick the coal/charcoal into it, then light it up.
It's a very neat mod, but i think we need a tutorial or wiki :-) 

For me this is solved!
